Automattic/wp-calypso
# Add provisioning error handler and register events

Related to #91779 
Based off of https://github.com/Automattic/wp-calypso/pull/92387

## Proposed Changes

* Tweak the provisioning component position to avoid being displayed over other elements.
* Register events.
* Handle error.

## Testing Instructions



* Make sure you have the `migration-flow/new-migration-instructions-step` feature flag enabled.
* Create a new site with the Import intent.
* Navigate until the migration instructions step.
* Check the provisioning position in the screen in different screen sizes.
* Update the link `if ( currentAction.status === 'error' ) {` to `if ( true ) {` to force displaying an error.
* Check that the screen handles the error.

## Screenshot

<img width="1271" alt="Screenshot 2024-07-10 at 17 12 59" src="https://github.com/Automattic/wp-calypso/assets/876340/9e7fa7ac-b28b-446e-bbbb-f8080309a043">

cc @fditrapani to validate how we are displaying the error. The message and link came from the existing instructions screen.

## Pre-merge Checklist
